userland/src/add_two_ints_client.cpp

```diff
@@ -0,0 +1,43 @@
+#include <algorithm>
+#include <chrono>
+#include <cmath>
+#include <iostream>
+#include <string>
+#include <sys/time.h>
+#include <thread>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <simple_msgs/AllBuiltinTypes.h>
+#include <simple_msgs/AllDynamicArrayTypes.h>
+#include <simple_msgs/AllPrimitiveTypes.h>
+#include <simple_msgs/AllStaticArrayTypes.h>
+#include <simple_msgs/Nested.h>
+#include <simple_msgs/String.h>
+#include <simple_msgs/Uint32.h>
+
+#include <userland_msgs/AddTwoInts.h>
+
+int main(int argc, char** argv)
+{
+  rclcpp::init(argc, argv);
+
+  auto node = rclcpp::Node::make_shared("add_two_ints_client");
+
+  auto client = node->create_client<userland_msgs::AddTwoInts>("add_two_ints");
```

Comment:
I modified the example to work asynchronously. Unfortunately, I can't hide the do { ... } loop because I can't spin the from inside the client.